tests/migration/online_migration/yast2_migration.pm

```diff
@@ -209,10 +209,15 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
-    # in leap to sle migration, we need click trust for package-hub's import-untrusted-gpg-key
-    if (is_leap_migration) {
+    # migration via smt will install packagehub and NVIDIA compute, we need click trust
+    # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
+    if ((get_var('SMT_URL') =~ /smt/) || (is_leap_migration)) {
         assert_screen 'import-untrusted-gpg-key', 60;
```

Comment:
this tag should be import-untrusted-gpg-key-phub ?

Reply:
> this tag should be import-untrusted-gpg-key-phub ?

No, this is a general tag though I added the packagehub needle to the tag.

Reply:
then why we need import-untrusted-gpg-key-nvidia ?

Reply:
NVIDIA compute module also will be installed here (on aarch64 and x86_64), need to import its gpg key. I added this tag because aarch64 worker's performance is not well somethimes, it caught the packagehub key needle twice though it had send_key 'alt-t'. If the worker's performance is not well, it cost time to continue after send key.

Reply:
I suppose following function can make sure the screen is updated, 
wait_screen_change(sub {
  send_key 'alt-t';
}, 15);
Or directly using assert_screen_change  to make sure screen change after type the key.